apps/thunderhub/docker-compose.yml

```
@@ -0,0 +1,27 @@
+version: "3.7"
+
+x-logging:
+  &default-logging
+  driver: journald
+  options:
+    tag: "umbrel-app {{.Name}}"
+
+services:
+  thub:
```

This should always be called `web` (was pointed out in the developer chat)
```suggestion
  web:
```